include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3625,8 +3647,12 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
     for (unsigned int comp = 0; comp < n_components; ++comp)
       for (unsigned int i = 0; i < dofs_per_component; ++i)
         operation.process_empty(values_dofs[comp][i]);
```

Comment:
If I see things correctly, this is the only place where we still use `n_vectorization_actual` (line 3646). Would it make sense to move the definition of line 3471 here?

Reply:
I have moved it down.

Reply:
I have one more request: I see that this loop here in lines 3646-3649 is exactly the same as the one in line 3604-3607. Can you move the former one out of the `if`? In that case, I would then further prefer to pull the computation into the `if, i.e., something like `if(std::count_if(...) < n_lanes)` (unless it looks very ugly).

Reply:
Done! I though normally you don't like the lambdas in if-statements ;)